src/components/families/family-form/student-form/StudentForm.tsx

```diff
@@ -1,13 +1,33 @@
 import React from "react";
 
-import { Button, Typography } from "@material-ui/core";
+import { Box, Button, IconButton, Typography } from "@material-ui/core";
+import { Add, RemoveCircle } from "@material-ui/icons";
+import { makeStyles } from "@material-ui/styles";
 
 import { StudentRequest } from "api/types";
 import StudentRole from "constants/StudentRole";
 import { DynamicField } from "types";
 
 import StudentFields from "../student-fields";
 
+const useStyles = makeStyles(() => ({
+  addButton: { marginLeft: 32, paddingLeft: 12, paddingRight: 12 },
+  addButtonIcon: { marginRight: 8 },
+  deleteButton: {
+    height: 16,
+    width: 16,
+  },
+  deleteButtonIcon: { height: 16, width: 16 },
+  roleLabel: {
+    width: 128,
+  },
```

Comment:
do we prefer longer lines for specifying styles or many short lines? Because I'm seeing
`addButton: { marginLeft: 32, paddingLeft: 12, paddingRight: 12 },`
when it could be
```
addButton: { 
  marginLeft: 32, 
  paddingLeft: 12, 
  paddingRight: 12 
},
```
but I'm also seeing
```
roleLabel: {
   width: 128,
},
```
when it could be
`roleLabel: { width: 128  },`

Reply:
good catch, this is totally up to personal preference but i like separating "list-ish" things by line! it makes diffs cleaner and makes it clear when an element was added in blames